### DISCUSSION TITLE
✳️ Support Outline by leaving a review ✳️

### DISCUSSION THREAD

**Discussion by tommoor** [outline/outline#6931] [General]
We don't really spend any time on advertising or marketing Outline (sad), but would love for more folks to learn about the project. 

If you've enjoyed using the platform and are willing to spend 10 minutes contributing back it would be super helpful to consider leaving a review on one of the popular software review sites. e.g.

- https://www.g2.com/products/outline/reviews
- https://www.capterra.com/p/210246/Outline
- https://www.producthunt.com/products/outline/reviews

Thanks folks! 🙏

**Comment by max-mykhailenko** [comment]
Add your productHunt ;) https://www.producthunt.com/products/outline
